README.md

```diff
@@ -48,6 +48,24 @@ It supports the following Azure credential types:
        ...
    </build>
    ```
+
```

Comment:
The title of this section is not very clear, change it to something like "how to use Azure credentials in your own Jenkins plugin"?

Reply:
done